xds/internal/balancer/edsbalancer/eds_impl.go

```diff
@@ -141,9 +141,9 @@ func (edsImpl *edsBalancerImpl) handleChildPolicy(name string, config json.RawMe
 			// TODO: (eds) add support to balancer group to support smoothly
```

Comment:
Could we rename the loop variable `id` to `lid` to make it explicit that we are dealing with a locationID here.

Reply:
Done